src/style.css

```
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  height: 15px;
+}
+
+.stars__item {
+  display: inline-block;
+  content: url(./images/star.svg);
+  margin: 0 1px;
+}
+
+.stars--1 :nth-child(-n+1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
```

```suggestion
.stars--5 .stars__item {
```
since there is only five stars, every one of them should be styled